### ISSUE TITLE
Android Builds failing on latest flutter Stable channel with v2.3.0

### ISSUE THREAD

**Issue by duttaoindril** [deepak786/intercom_flutter#96]
```
e: .pub-cache/hosted/pub.dartlang.org/intercom_flutter-2.2.1/android/src/main/kotlin/io/maido/intercom/IntercomFlutterPlugin.kt: (17, 7): Redeclaration: IntercomFlutterPlugin
e: .pub-cache/hosted/pub.dartlang.org/intercom_flutter-2.3.0/android/src/main/kotlin/io/maido/intercom/IntercomFlutterPlugin.kt: (20, 7): Redeclaration: IntercomFlutterPlugin
e: .pub-cache/hosted/pub.dartlang.org/intercom_flutter-2.3.0/android/src/main/kotlin/io/maido/intercom/IntercomFlutterPlugin.kt: (28, 58): No value passed for parameter 'application'
e: .pub-cache/hosted/pub.dartlang.org/intercom_flutter-2.3.0/android/src/main/kotlin/io/maido/intercom/IntercomFlutterPlugin.kt: (30, 65): No value passed for parameter 'application'
e: .pub-cache/hosted/pub.dartlang.org/intercom_flutter-2.3.0/android/src/main/kotlin/io/maido/intercom/IntercomFlutterPlugin.kt: (37, 36): Unresolved reference: NonNull
e: .pub-cache/hosted/pub.dartlang.org/intercom_flutter-2.3.0/android/src/main/kotlin/io/maido/intercom/IntercomFlutterPlugin.kt: (39, 56): No value passed for parameter 'application'
e: .pub-cache/hosted/pub.dartlang.org/intercom_flutter-2.3.0/android/src/main/kotlin/io/maido/intercom/IntercomFlutterPlugin.kt: (41, 63): No value passed for parameter 'application'
e: .pub-cache/hosted/pub.dartlang.org/intercom_flutter-2.3.0/android/src/main/kotlin/io/maido/intercom/IntercomFlutterPlugin.kt: (207, 38): Unresolved reference: NonNull

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':intercom_flutter:compileDebugKotlin'.
> Compilation error. See log for more details

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 29s
Exception: Gradle task assembleDebug failed with exit code 1
Exited (sigterm)
```

**Comment by julrach** [comment]
Getting the same issue!!!

**Comment by julrach** [comment]
What versions can be used while waiting for https://github.com/v3rm0n/intercom_flutter/pull/97? We are in the middle of a release and this is sadly holding us back. :(

@Zazo032 @v3rm0n

**Comment by Zazo032** [comment]
Should be fixed in `2.3.1`
